ui/core.js

```diff
@@ -126,6 +128,67 @@ $.fn.extend( {
 				$( this ).removeAttr( "id" );
 			}
 		} );
+	},
+
+	form: function() {
```

Comment:
I think this can be simplified to:

```js
var form = this.prop( "form" );
return typeof form === "string" ? this.closest( "form" ) : $( form );
```